python/lsst/ts/ddsconfig/utils.py

```diff
@@ -28,11 +28,16 @@ def get_pkg_root() -> pathlib.Path:
     return pathlib.Path(__file__).resolve().parents[4]
 
 
+def get_data_dir() -> pathlib.Path:
+    """Return the data directory of this package."""
+    return pathlib.Path(__file__).resolve().parents[0] / "data"
```

Comment:
Is this the recommended approach? I'm very relieved, if so. It seems much simpler than having to call special functions, e.g. from setuptools.

Reply:
You, Tiago and me discussed this on Slack and decided that we wanted to do it like this since pyproject.toml requires all data files to be inside the package.